IsraelHiking.Web/src/application/services/image-attribution.service.ts

```diff
@@ -16,6 +16,12 @@ export class ImageAttributionService {
     private readonly resources = inject(ResourcesService);
     private readonly httpClient = inject(HttpClient);
 
+    private extractPlainText(html: string): string {
+        const parser = new DOMParser();
+        const doc = parser.parseFromString(html, 'text/html');
+        return doc.documentElement.textContent.replace(/( *\n *)+/g, '\n').replace(/ +/g, ' ').trim();
```

Comment:
Should this be `\s` instead of " " (regular space)?

Reply:
`\s` includes `\n` and I wanted to preserve them using `replace(/( *\n *)+/g, '\n')`